Fakebook.Profile/Fakebook.Profile.RestApi/Startup.cs

```diff
@@ -17,36 +19,92 @@ namespace Fakebook.Profile.RestApi
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration) {
+        public Startup(IConfiguration configuration, IWebHostEnvironment env)
+        {
             Configuration = configuration;
+            _env = env;
         }
 
         public IConfiguration Configuration { get; }
+        private readonly IWebHostEnvironment _env;
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services) {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors(options =>
+            {
+                options.AddDefaultPolicy(
+                    builder =>
+                    {
+                        builder.WithOrigins("https://fakebook.revaturelabs.com/")
```

Comment:
this cors config isn't useful, useful would be to allow localhost 4200.
this api will never receive requests from pages served by revaturelabs.net unless this api is also itself running on that same origin.